index.html

```diff
@@ -28,11 +41,31 @@ <h3>Add books</h3>
                 <input type="text" required placeholder="Author" id="book_author">
                 <br>
                 <br>
-                <input type="submit" value="Add" id="addButton">
+                <button type="submit" form="form" value="Submit" class="btn btn-success">Add</button>
```

Comment:
- [ ] OPTIONAL
Kindly note that the add button is **shorter** and it's **aligned to the right** with the form inputs. Kindly endeavor to improve upon this aspect.

**Your forms page:**
![image](https://user-images.githubusercontent.com/12745474/138119292-8815b767-2914-4c59-a9f8-039560eb78ae.png)

**The wireframe forms page:**
![image](https://user-images.githubusercontent.com/12745474/134705307-2184f998-9c02-4279-ac18-7de5d37e50ab.png)